src/theme/global.css

```diff
@@ -14,3 +14,46 @@
 mark.current {
     background-color: orange;
 }
+
+.tab-bar {
+    margin: 8px;
+    margin-top: 0;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    border-radius: 5px;
+}
+
+.action-button {
+    --border-radius: 5px;
+    font-size: large;
+    height: 60px;
```

Comment:
i think this should be removed, and it should rely on the content to set the height.